File: backend2/DataAccess/DaoGenerated/システム共有/tm_afkikanDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             医療機関マスタ
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afkikanDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_afkikan";
		public string kikancd { get; set; }                       //医療機関コード（自治体独自）
		public string hokenkikancd { get; set; }                  //保険医療機関コード
		public string kanakikannm { get; set; }                   //医療機関名カナ
		public string kikannm { get; set; }                       //医療機関名
		public string yubin { get; set; }                         //郵便番号
		public string adrs { get; set; }                          //住所
		public string? katagaki { get; set; }                     //方書
		public string? tel { get; set; }                          //電話番号
		public string? fax { get; set; }                          //FAX番号
		public string? syozokuisikai { get; set; }                //所属医師会
		public bool stopflg { get; set; }                         //使用停止フラグ
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/JbdGjsService/Db/DaoGenerated/システム共有/tm_afkikanDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             医療機関マスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tm_afkikanDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afkikan"
        Public Property kikancd As String                       '医療機関コード（自治体独自）
        Public Property hokenkikancd As String                  '保険医療機関コード
        Public Property kanakikannm As String                   '医療機関名カナ
        Public Property kikannm As String                       '医療機関名
        Public Property yubin As String                         '郵便番号
        Public Property adrs As String                          '住所
        Public Property katagaki As String                     '方書
        Public Property tel As String                          '電話番号
        Public Property fax As String                          'FAX番号
        Public Property syozokuisikai As String                '所属医師会
        Public Property stopflg As Boolean                         '使用停止フラグ
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
